mne/viz/backends/_qt.py

```diff
@@ -435,6 +436,24 @@ def _window_ensure_minimum_sizes(self, sz):
             self._process_events()
             self._process_events()
 
+    def _window_set_theme(self, theme):
+        if theme == 'auto':
+            import darkdetect
```

Comment:
Make sure optional dependencies stay optional, e.g.
```suggestion
            try:
                import darkdetect
            except Exception:
                detected_theme = None
            else:
                detected_theme = darkdetect.theme()
            theme = 'light' if detected_theme is None else deteced_theme.lower()
```
Something similar should be done for `qdarkstyle`